src/components/WorkHistory/index.tsx

```diff
@@ -93,13 +121,26 @@ const WorkplaceCard: FC<IWorkplace> = ({
   );
 };
 
-const WorkplaceCardEdit: FC<IWorkplace> = ({
+interface IWorkplaceEdit {
+  workplaceId: number;
+  role: string;
+  nameValue: string;
+  description: string;
+  startDate: string;
+  endDate: string;
+  fetchAllOrganisationsData: any;
+}
+
+type SelectOptionType = { label: string; value: string };
+
+const WorkplaceCardEdit: FC<IWorkplaceEdit> = ({
   workplaceId,
   role,
   nameValue,
   description,
   startDate,
   endDate,
+  fetchAllOrganisationsData,
```

Comment:
The name is not good. It looks like a function for fetching data but it's an options array of object.